app/dependencies.js

```diff
@@ -28,25 +28,75 @@ module.exports = {
         npm: [
             'dustjs-linkedin@~2.6.1',
             'dustjs-helpers@~1.6.1',
+            "engine-munger@^0.2.5",
             'adaro@^0.1.5'
         ],
         npmDev: [
             'grunt-dustjs@^1.2.1'
-        ]
+        ],
+        tasks: function (options) {
+            if (!options.i18n) {
+                return "dustjs";
+            }
+        },
+        templates: "dustjs/**"
     },
 
     i18n: {
+        npm: function (options) {
+            if (options.templateModule === 'dustjs') {
```

Comment:
Main reason I haven't: I want to refactor so that many engines are available. This bit's making me feel the main and stay aware of that.